model/atmosphere/dycore/src/icon4py/model/atmosphere/dycore/velocity_advection.py

```diff
@@ -44,10 +45,12 @@ def __init__(
         vertical_params: v_grid.VerticalGrid,
         edge_params: grid_states.EdgeParams,
         owner_mask: fa.CellField[bool],
-        backend: gtx_backend.Backend | None,
+        backend: gtx.backend.Backend | Literal["gpu", "cpu"] | dict[str, Any] | None = None,
     ):
         self.grid: icon_grid.IconGrid = grid
-        self._backend = backend
+        self._backend = (
+            backend if isinstance(backend, gtx.backend.Backend) else customize_backend(**backend)
+        )
```

Comment:
Why this different pattern here compared to diffusion/solve_nonhydro? Is it a refactoring artifact?

Reply:
left-over. thanks for pointing out